AFNetworking/AFHTTPClientSKZ.m

```diff
@@ -674,6 +674,19 @@ - (void)enqueueBatchOfHTTPRequestOperations:(NSArray *)operations
     [self.operationQueue addOperation:batchedOperation];
 }
 
++ (dispatch_queue_t)backgroundQueue
+{
+    static dispatch_once_t pred;
+    static dispatch_queue_t backgroundQ;
+    
+    dispatch_once(&pred, ^{
+        backgroundQ = dispatch_queue_create("background_http_queue", DISPATCH_QUEUE_SERIAL);
```

Comment:
Oh, that makes sense. Although, since we likely have some code that may depend on completion blocks firing on the main queue, would it make sense to handle this inside the Core Data Manager, rather than inside AFNetworking? In the same vein, the previous writeToTemporaryContext executed completion blocks on the main queue, does your updated form do this as well?